src/components/Header.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { View, Text, SafeAreaView, StyleSheet } from 'react-native';
+
+import { theme } from '../constants';
+const { SIZES, FONTS, COLORS } = theme;
+
+const Header = ({ label }) => {
+    return (
+        <>
+            <SafeAreaView/>
+            <View style={styles.labelContainer}>
+                <Text style={styles.labelText}>{label}</Text>
+            </View>
+        </>
```

Comment:
The CodeClimate tells them that you can try refactoring here 😊 

![image](https://user-images.githubusercontent.com/69128578/114465172-5a80c200-9bac-11eb-97d4-8a86748081f5.png)

Reply:
Ready, @JacquelineFM , we have solved the problems with CodeClimate.

Reply:
Excelent!